ports/espressif/boards/sunton_esp32_2424S012/pins.c

```diff
@@ -0,0 +1,42 @@
+#include "shared-bindings/board/__init__.h"
+#include "shared-module/displayio/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    // User button
+    { MP_ROM_QSTR(MP_QSTR_BOOT0), MP_ROM_PTR(&pin_GPIO9) },
+    { MP_ROM_QSTR(MP_QSTR_BUTTON0), MP_ROM_PTR(&pin_GPIO9) },
+
+    // IO8. This is on the schematic linked to a 10k pull up and "S1"
+    // No further indication of what "S1" is or what this pin is for
+    { MP_ROM_QSTR(MP_QSTR_IO8), MP_ROM_PTR(&pin_GPIO8) },
```

Comment:
In this picture, from https://www.luisllamas.es/en/sunton-esp32-round/, I see a switch, maybe that's it?
![image](https://github.com/adafruit/circuitpython/assets/2847802/fa17ac09-4eb5-46cf-981a-d8163c0e3cb5)